src/NHibernate.Test/Async/SystemTransactions/DistributedSystemTransactionFixture.cs

```diff
@@ -357,6 +460,60 @@ protected override void OnTearDown()
 		}
 
 		[Theory]
+		[Description(@"Two session in two txscope
+ (without an explicit NH transaction)
+ and with a rollback in the second dtc and a ForceRollback outside nh-session-scope.")]
+		public void TransactionInsertLoadWithRollBackTask(bool explicitFlush)
```

Comment:
Duplicated synchronous test.

This are the one called by `MultiThreadedTransaction` test, but there should not be copied here in async, since MultiThreadedTransaction is not there either. It looks like a regression of the async generator.

Reply:
It's a bug, thanks for discovering it. By ignoring the `MultiThreadRunner` class, the calculation for the method conversion fails to calculate the the correct conversion in this case. [I've made](https://github.com/maca88/AsyncGenerator/issues/69) an issue, with a simple test case demonstrating the bug. The old version would also not work in this case. In order to fix it I will have to revisit the method conversion  logic and find a better way to calculate the final conversion. I will post here once it is fixed.

Reply:
With the version `0.6.1`, the mentioned synchronous tests are not generated anymore.

Reply:
Thanks @maca88